### PR TITLE
apache: fix return value for silent_status

### PR DESCRIPTION
**Pull request by vvidic**

### CODE REVIEW

**Comment by vvidic** [comment]
```
Running: ocft test -v apache
Initializing 'apache' ...
Done.

Starting 'apache' case 0 'check base env':
ERROR: './apache monitor' failed, the return code is 2.
Starting 'apache' case 1 'check base env: set non-existing OCF_RESKEY_statusurl':
ERROR: './apache monitor' failed, the return code is 2.
Starting 'apache' case 2 'check base env: set non-existing OCF_RESKEY_configfile':
ERROR: './apache monitor' failed, the return code is 2.
Starting 'apache' case 3 'normal start':
ERROR: './apache monitor' failed, the return code is 2.
Starting 'apache' case 4 'normal stop':
ERROR: './apache monitor' failed, the return code is 2.
Starting 'apache' case 5 'double start':
ERROR: './apache monitor' failed, the return code is 2.
Starting 'apache' case 6 'double stop':
ERROR: './apache monitor' failed, the return code is 2.
Starting 'apache' case 7 'running monitor':
ERROR: './apache monitor' failed, the return code is 2.
Starting 'apache' case 8 'not running monitor':
ERROR: './apache monitor' failed, the return code is 2.
Starting 'apache' case 9 'unimplemented command':
ERROR: './apache monitor' failed, the return code is 2.
```

**Comment by vvidic** [comment]
```
+ apache_monitor
+ silent_status
+ local pid
+ local rc=false
+ local retries=0
+ [ monitor = monitor ]
+ ocf_is_probe
+ [ monitor = monitor -a 0 = 0 ]
+ true
+ get_pid
+ [ -f /var/run/apache2/apache2.pid ]
+ false
+ pid=
+ [ -n  ]
+ : No pid file
+ [ 0 -le 0 ]
+ break
+ return false
/usr/lib/ocf/resource.d/heartbeat/apache: 159: return: Illegal number: false
```

File: heartbeat/apache [hunk 1]
```diff
@@ -131,7 +131,7 @@ ProcessRunning() {
 }
 silent_status() {
 	local pid
-	local rc=false
+	local rc=1
```
Comment (oalbrigt) [inline heartbeat/apache]: Can you update this to`$OCF_ERR_GENERIC`?

**Comment by oalbrigt** [comment]
LGTM. Thanks.

**Comment by krig** [comment]
Great, thanks!

**Comment by HideoYamauchi** [comment]
Thanks!

**Pull request merged**